.github/actions/publish_anaconda/action.yml

```diff
@@ -0,0 +1,27 @@
+name: Publish Anaconda
+description: Publish a noarch python package to anaconda.org
+inputs:
+  anaconda_token:
+    required: true
+    description: The anaconda token to use for upload
+
+  dist_path:
+    required: true
+    description: A path to the python dist tarball to be uploaded
+
+runs:
+  using: composite
+  steps:
+    - name: Setup Miniconda
+      uses: conda-incubator/setup-miniconda@v2.2.0
+      with:
+        activate-environment: ""
+        auto-activate-environment: true
```

Comment:
Not known, perhaps it should be `auto-activate-base` (see https://github.com/marketplace/actions/setup-miniconda#activate-base-environment)

Reply:
Terrific spot! After a bit of further reading it would seem I don't even need to activate the base environment.